pages/common/git-graft.md

```diff
@@ -0,0 +1,9 @@
+# git graft
+
+> Merge commits from a specific branch into another branch and delete the source branch.
+> Part of `git-extras`.
+> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-graft>.
+
+- Merge all commits not present on the target branch from the source branch, and delete the source branch:
```

Comment:
This makes it longer, but re-reading it, without this extra bit the sentence sounds incomplete.
```suggestion
- Merge all commits not present on the target branch from the source branch to target branch, and delete the source branch:
```

Reply:
Broken down it would be `merge sth from sth`. To me it doesn't sound that bad.

Reply:
Doesn't matter, with the three extra words it's definetely clearer.

Reply:
What does `sth` mean there, @marchersimon? I'm confused.

Reply:
Isn't `sth` a common abbreviation of `something` and a placeholder if you want to describe how a verb can be used? (At least that's how most dictionaries do it.)

Reply:
DId someone unresolve this or did I not commit it?

Reply:
I think @bl-ue unresolved. You committed it in e0298f2.

Reply:
Ah @marchersimon I see!  I've not seen `sth` yet, I've only seen `smth` 🤔